src/pymortests/strategies.py

```diff
@@ -382,7 +382,7 @@ def base_vector_arrays(draw, count=1, dtype=None, max_dim=100):
     length = space.dim
 
     # this lets hypothesis control np's random state too
-    random = draw(hyst.randoms())
+    random = draw(hyst.random_module())
```

Comment:
@renefritze, what exactly does this do? Are you aware that pyMOR also manages it's own random state?

Reply:
- https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.random_module
 - Yes. 
 - Also: https://github.com/pymor/pymor/pull/910/commits/f7fd7671c05ca53ed1c464b60b1c10e0b27bfa8a

Reply:
> Also: f7fd767

Yeah but I was referring to `pymor.tools.random`. This won't be managed by hypothesis. However, at least for the demo tests, the state is reset before each demo execution in `pymortests.demos`.

Reply:
Yes. I know. The vectorarray random tests explicitly manage that too.

Reply:
Ah, actually only somewhat though.

Reply:
Might be possible to let hypothesis manage our default random state as well. However, not in this PR and maybe only after it has proven to be necessary ..